src/docs/workflows/device-commissioning.mdx

```diff
@@ -0,0 +1,118 @@
+import { Divider, TOC } from '../../app/components';
+
+# Device Commissioning Workflow
+
+> This guide is currently a draft. It is under review and subject to changes in the future.
+> WIP: Include more images and screens
+
+When using an application that deals with physical devices and their data, one of the first challenges that a user will face is how to get those devices connected to the application and properly configured.
+
+Depending on the particular devices and the nature of your application, this process can vary significantly. This workflow describes some of the common steps and elements of connecting and configuring devices in order to create as much consistency as possible between different applications.
+
+<TOC
+    anchors={[
+        { title: 'Introduction', hash: '#device-commissioning-workflow' },
+        { title: 'Steps', hash: '#steps' },
+        { title: 'Design Variations', hash: '#design-variations' },
+        { title: 'Design Tricks', hash: '#design-tricks' },
+    ]}
+/>
+
+<Divider />
+
+# Steps
+
+## Setting Up Devices One-by-One
+
+Setting up devices typically involves two phases: Device Discovery and Device Configuration.
+
+### Device Discovery
+
+The search phase generally consists of the following steps (some steps may not be required for your application):
+
+-   **Prepare the device**: provide instructional text, [images, and animations](/style/illustrations) to the user so they can ensure that the device is discoverable. Some example instructions might include "move within range of the device," "turn off all other devices," "press the XYZ button on the device," "scan the code on the device using your camera," etc.
+-   **Select the device type**: if there could be multiple discoverable devices, you may wish to allow your user to narrow down the options prior to discovery. This is also an important step if different devices have different steps and protocols for connecting and / or configuring.
+-   **Identify the target device**: If multiple devices are discovered, provide the user a way to choose the target device from the list. To assist with identification (particularly if devices are not readily identifiable by their name or id), you can implement an indicator light to help the user locate the target device.
+-   **Confirm device selection**: have the user confirm that this is the device they are looking for. On this screen, you would show device details (serial number, address, model, etc.) for the user to double-check their selection.
+-   **Confirm device's functionality**: To verify that the device is working correctly before entering the configuration phase, you may wish to give the user the ability to test the functionality. For example, you could give them the option to turn on or off a light bulb or allow them to press a button on the device and see some status change in the application.
```

Comment:
Can we get some consistency on capitalization for each bullet point after the `**blah blah**:`. Some are capitalized and some aren't... not sure which one is appropriate, but it should be consistent.